### ISSUE TITLE
Wordreference.com: features not working on Kiwi browser

### ISSUE THREAD

**Issue by alleluia20** [kiwibrowser/src.next#326]
When using the language dictionary website https://www.wordreference.com/, there are at least two features not working on Kiwi browser:

1) On other browsers, when you start typing in the search field, a list of suggestions opens for possible entries in both ways (I mean: if you have chosen "English to Spanish", and you start typing on the search field, suggestions of entries are shown, both for English to Spanish and for Spanish to English).
On Kiwi browser, no suggestions show.

2) On other browsers, when you are reading a dictionary entry, you can click on a word to browse directly to that word. For example, let us suppose you are reading "be" in the English to Spanish dictionary. One possible translation (the main one, actually) is "ser". Then, if you tap on the word "ser", it opens the Spanish to English dictionary for the word "ser".
On Kiwi browser, tapping on a word does nothing.

Regards.

**Comment by DUOLabs333** [comment]
Number 1 works for me, I'm not completely sure what you mean by 2, but clicking on words do lead me to their respective pages. Do you have the builtin ad blocker enabled?

**Comment by alleluia20** [comment]
Hello,

I see that number 2 is working now. It may have been fixed in a recent
update, I do not know...

Unfortunately, number 1 is not working for me. I have the built-in
adblocker disabled. I have uBlock, but the problem persists after disabling
all the extensions (including uBlock).

Regards.

**Comment by DUOLabs333** [comment]
Are you using the Play Store version or the GitHub version?

**Comment by alleluia20** [comment]
The Play Store version of Kiwi: version 93.0.4577.39

**Comment by DUOLabs333** [comment]
So if you go to the home page and start typing, no suggestions come up?

**Comment by alleluia20** [comment]
That is correct, unfortunately. No suggestions, either on the home page or
in the search field when reading a dictionary entry.

**Comment by DUOLabs333** [comment]
Do you have any settings or extensions that are not the default?

**Comment by alleluia20** [comment]
Regarding extensions, the problem persists after disabling them all.

As for the settings, I think I changed something, but not much. I cannot
remember exactly... The built-in adblocker is disabled. If you want me to
check a specific setting, tell me which setting you have in mind.

Regards.

**Comment by DUOLabs333** [comment]
Oh, you're right it does fail. There must have been a regression (I was on the previous version).

**Comment by DUOLabs333** [comment]
I upgraded now, and it still gives me autocomplete. Could you put a short video to show case it? Lite mode isn't on is it?

**Comment by alleluia20** [comment]
I am quite busy these days but, anyway, there is nothing to be shown on a video... When I type on the search field, it is static, the typed characters are displayed, but nothing unfolds.

I mean, if you had never used this site, you would not notice anything strange. There is nothing to show on a video: the problem is not something happening, the problem is what does not happen.

**Comment by alleluia20** [comment]
Ha, ha, the auto complete fold-up menu shows after disabling the built-in
"hide ads" option in that site.

I have had several rendering issues since I started using Kiwi some months
ago, and I think that all of them have been fixed by disabling this option.

Having uBlock Origin available, and with the built-in ad blocker causing so
many false positives: have you considered disabling the latter by default?
(it is not possible to disable it globally, not even manually, see my
bug #440).

Regards.
